manifests/base/jupyter-rocm-pytorch-notebook-imagestream.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com/red-hat-data-services/notebooks/blob/main/jupyter/rocm/pytorch"
+    opendatahub.io/notebook-image-name: "ROCm-PyTorch"
+    opendatahub.io/notebook-image-desc: "Jupyter ROCm optimized PyTorch notebook image for ODH notebooks."
+    opendatahub.io/notebook-image-order: "14"
+  name: jupyter-rocm-pytorch
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+    # N Version of the image
+    - annotations:
+        opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.9"},{"name":"ROCm-PyTorch","version":"2.3"}]'
+        opendatahub.io/notebook-python-dependencies: '[{"name":"ROCm-PyTorch","version":"2.3"},{"name":"Tensorboard","version":"2.16"},{"name":"Kafka-Python","version":"2.0"},{"name":"Matplotlib","version":"3.8"},{"name":"Numpy","version":"1.26"},{"name":"Pandas","version":"2.2"},{"name":"Scikit-learn","version":"1.4"},{"name":"Scipy","version":"1.12"},{"name":"Elyra","version":"3.16"},{"name":"PyMongo","version":"4.6"},{"name":"Pyodbc","version":"5.1"}, {"name":"Codeflare-SDK","version":"0.19"}, {"name":"Sklearn-onnx","version":"1.16"}, {"name":"Psycopg","version":"3.1"}, {"name":"MySQL Connector/Python","version":"8.3"}]'
+        openshift.io/imported-from: quay.io/modh/rocm-notebooks
+        opendatahub.io/workbench-image-recommended: 'true'
+        opendatahub.io/notebook-build-commit: $(odh-rocm-pytorch-notebook-image-commit-n)
+      from:
+        kind: DockerImage
+        name: $(odh-rocm-pytorch-notebook-image-commit-n)
```

Comment:
```suggestion
        name: $(odh-rocm-pytorch-notebook-image-n)
```

Reply:
The commit will have to stay. As seen in the others https://github.com/red-hat-data-services/notebooks/blob/5923bd64807ae255fc4b24e3a31ca695d5f7caf6/manifests/base/jupyter-minimal-gpu-notebook-imagestream.yaml#L24

Reply:
But this is a different field. Check the line 27 in that file you reference.

Please, remove the `commit` infix from the two lines I commented. I also can see that it is not correct for the rocm files in upstream repo.